packages/evm/contracts/disputes/DisputeManager.sol

```diff
@@ -1126,34 +1104,102 @@ contract DisputeManager is ControlledRecoverable, ICRVotingOwner, IDisputeManage
+    */
+    function _getDispute(uint256 _disputeId) internal view returns (Dispute storage) {
         require(_disputeId < disputes.length, ERROR_DISPUTE_DOES_NOT_EXIST);
+        return disputes[_disputeId];
     }
 
     /**
-    * @dev Internal function to check if a certain dispute round exists, it reverts if it doesn't
-    * @param _disputeId Identification number of the dispute to be checked
-    * @param _roundId Identification number of the dispute round to be checked
+    * @dev Internal function to fetch a dispute and round
+    * @param _disputeId Identification number of the dispute to be fetched
+    * @param _roundId Identification number of the round to be fetched
+    * @return dispute Dispute instance requested
+    * @return round Round instance requested
     */
-    function _checkRoundExists(uint256 _disputeId, uint256 _roundId) internal view {
-        _checkDisputeExists(_disputeId);
-        require(_roundId < disputes[_disputeId].rounds.length, ERROR_ROUND_DOES_NOT_EXIST);
+    function _getDisputeAndRound(uint256 _disputeId, uint256 _roundId)
+        internal
+        view
+        returns (Dispute storage dispute, AdjudicationRound storage round)
+    {
+        dispute = _getDispute(_disputeId);
+        _checkRoundExists(dispute, _roundId);
+        round = dispute.rounds[_roundId];
+    }
+
+    /**
+    * @dev Internal function to fetch a dispute with its config
+    * @param _disputeId Identification number of the dispute to be fetched
+    * @return dispute Dispute instance requested
+    * @return config Protocol config used for the given dispute
+    */
+    function _getDisputeWithConfig(uint256 _disputeId)
+        internal
+        view
+        returns (Dispute storage dispute, Config memory config)
+    {
+        dispute = _getDispute(_disputeId);
+        config = _getDisputeConfig(dispute);
+    }
+
+    /**
+    * @dev Internal function to fetch a dispute and round with its config
+    * @param _disputeId Identification number of the dispute to be fetched
+    * @param _roundId Identification number of the round to be fetched
+    * @return dispute Dispute instance requested
+    * @return round Round instance requested
+    * @return config Protocol config used for the given dispute
+    */
+    function _getDisputeAndRoundWithConfig(uint256 _disputeId, uint256 _roundId)
+        internal
+        view
+        returns (Dispute storage dispute, AdjudicationRound storage round, Config memory config)
+    {
+        dispute = _getDispute(_disputeId);
+        _checkRoundExists(dispute, _roundId);
+        round = dispute.rounds[_roundId];
+        config = _getDisputeConfig(dispute);
+    }
+
+    /**
+    * @dev Internal function to get the Protocol config used for a dispute
+    * @param _dispute Dispute querying the Protocol config of
+    * @return Protocol config used for the given dispute
+    */
+    function _getDisputeConfig(Dispute storage _dispute) internal view returns (Config memory) {
```

Comment:
Nitpick: maybe it's worth pushing this up to be after `_getDispute()` and `_getDisputeRound()` so all the "singular" versions live together :).